skgstat/Variogram.py

```diff
@@ -14,10 +14,9 @@
 from skgstat import estimators, models, binning
 from skgstat import plotting
 from skgstat.util import shannon_entropy
-from .MetricSpace import MetricSpace
+from .MetricSpace import MetricSpace, MetricSpacePair, ProbabalisticMetricSpace
```

Comment:
Do we actually need the import of MetricSpacePair here? I think it is not used anywhere in this file...

Reply:
It is used here:

https://github.com/mmaelicke/scikit-gstat/blob/probabalistic-metric-space/skgstat/Variogram.py#L246

just so you can do `Variogram(..., samples=1234)`